Unity-Technologies/UnityRenderStreaming
# feat: Add gyrosample

This pull request adds the sample which integrating gyro input.
This sample can run on iOS only. 
![unnamed](https://user-images.githubusercontent.com/1132081/107727299-4f392380-6d2e-11eb-9e8c-770ba88d07e0.gif)

### How to test
1. Launch the signaling server using public mode. Please refer to [this link](webapp.md) for launching.
2. Open `Gyro` scene and specify IP address of the signaling server to the `Signaling URL` parameter on the inspector.
3. Specify the build platform (Currently iOS supported only) and start building. Connect the device to the PC, and transfer the application to the device.
4. Open `Broadcast` scene and Specify IP address of the signaling server to the `Signaling URL` parameter on the inspector.
5. Click on the Play button in Unity Editor.
6. Push the `Play` button on the device and check streaming video.
